images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
Moving to SCL ruby or not isn't specific to container, as it also affects VM appliances and should be discussed with other people as well...

Can we split this out for now by maintaining our own ruby container, and consider using stock ruby container separately?